src/index.js

```diff
@@ -1,30 +1,21 @@
 import './style.css';
-import STORE from './Modules/store.js';
+import gamelist from './Modules/score.js';
+import games from './Modules/store.js';
 
-const { v4: uuidv4 } = require('uuid');
+const btn = document.querySelector('.btn');
+const name = document.querySelector('#name');
+const score = document.querySelector('#score');
+const submit = document.querySelector('#submit');
 
-const pieces = [
-  {
-    name: 'Joy',
-    score: '200',
-    id: uuidv4(),
-  },
+function list(i) {
```

Comment:
## Optional: Please Open this link 👉 [Arrow Function](https://www.w3schools.com/react/react_es6_arrow.asp) to learn more. make sure to implement the change. 
- **Why Arrow Function**: One of the most heralded features in modern JavaScript is the introduction of arrow functions, sometimes called 'fat arrow' functions, utilizing the new token `=>` . These functions two major benefits - a very clean concise syntax and more intuitive scoping and this binding.